atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java

```diff
@@ -182,13 +182,13 @@ InetSocketAddress getAddressForHost(String host) throws UnknownHostException {
                 .filter(predicate)
                 .collect(Collectors.toSet());
         if (filteredHosts.isEmpty()) {
-            log.error("No hosts match the provided predicate.");
+            log.info("No hosts match the provided predicate.");
```

Comment:
Suggestion for a quick improvement, while we're here: should also log the predicate, if plausible.